test/associations.spec.js

```diff
@@ -47,25 +49,6 @@ describe('associations', function () {
                     });
                 });
         });
-        it('should be able to dissociate', function (done) {
```

Comment:
Spoke to @kristofsajdak reverse linking tests have been removed because the implementation if flawed and we have no use for them right now